README.md

```diff
@@ -18,6 +18,8 @@
 
 PcapPlusPlus enables decoding and forging capabilities for a large variety of network protocols. It also provides easy to use C++ wrappers for the most popular packet processing engines such as [libpcap](https://www.tcpdump.org/), [WinPcap](https://www.winpcap.org/), [Npcap](https://nmap.org/npcap/), [DPDK](https://www.dpdk.org/), [eBPF AF_XDP](https://www.kernel.org/doc/html/next/networking/af_xdp.html) and [PF_RING](https://www.ntop.org/products/packet-capture/pf_ring/).
 
+Translations: [正體中文](./translation/README-zh-tw.md)
```

Comment:
All translations will be here.

Reply:
What about this pattern ? https://github.com/jonatasemidio/multilanguage-readme-pattern/tree/master

Reply:
@clementperon  My concern is that there are many documents, e.g. CONTRIBUTING.md (, which is at the root), and we will also have translation files for them. If we follow this pattern, the root directory will have a lot of translation files.